packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -85,16 +85,21 @@ function MaybeIframe( {
 export default function VisualEditor( { styles } ) {
 	const {
 		deviceType,
+		isActive,
```

Comment:
Should this be more something like `isWelcomeGuideVisible`?

Reply:
Changed.